assemblies/lib/pom.xml

```diff
@@ -129,17 +129,6 @@
         </exclusion>
       </exclusions>
     </dependency>
-
-    <dependency>
-      <groupId>pentaho</groupId>
-      <artifactId>pentaho-mongo-utils</artifactId>
-      <version>${pentaho-mongodb-plugin.version}</version>
-    </dependency>
-    <dependency>
```

Comment:
I don't think we can remove the mongo java driver from data-integration/lib; the dataservice-server-plugin in karaf needs this dependency to be on the main classpath for now.